packages/teams-js/src/public/interfaces.ts

```diff
@@ -711,6 +711,14 @@ export interface Context {
    */
   userClickTime?: number;
 
+  /**
```

Comment:
Yeah, I have kinda the same question tbh. Finally, I opted to added it as well here because I have noticed that some apps do use the legacy context in some way or another, so it was just to add this field here as well so it can be accessed always.

Reply:
So yeah, this must be mapped in the legacy context. Apparently, the host answers back with the context in the legacy format and then gets mapped to the new structure before passing it to the app:
![image](https://github.com/user-attachments/assets/036b9bcb-b363-419a-a09b-af959eda1ca4)
So resolving this as it must be mapped to appear in the function.